pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleJsonServlet.java

```diff
@@ -158,12 +158,6 @@ protected boolean isValidResource(@Nonnull SlingHttpServletRequest request, @Non
             moduleMap.put(VIEW_URI, view_uri);
         }
 
-        // Process view_uri
-        if (System.getenv("PORTAL_URL") != null) {
```

Comment:
So, this will NOT be appearing in the API anymore?
Just confirming...

Reply:
line 161 is a duplicated code block.